apps/meteor/server/startup/migrations/v293.ts

```diff
@@ -0,0 +1,97 @@
+import { addMigration } from '../../lib/migrations';
+import { upsertPermissions } from '../../../app/authorization/server/functions/upsertPermissions';
+
+addMigration({
+	version: 293,
+	async up() {
+		await upsertPermissions();
+
+		// TODO: Please comment this out, for next major release
+
```

Comment:
Is this necessary? If not remove the commentary

Reply:
this thing was supposed to be gone 🤦🏼